frontend/src/features/mapSideBar/SidebarContents/subforms/BuildingReviewPage.tsx

```diff
@@ -76,8 +76,8 @@ export const BuildingReviewPage: React.FC<any> = (props: IReviewProps) => {
                     className="edit"
                     onClick={() =>
                       setEditInfo({
-                        ...defaultEditValues,
-                        identification: !editInfo.identification,
+                        ...editInfo,
+                        identification: formikProps.isValid && !editInfo.identification,
```

Comment:
Do not allow the section to be placed in read only mode if there are validation errors as those errors are only displayed in edit mode.